### DISCUSSION TITLE
跳转子应用问题

### DISCUSSION THREAD

**Discussion by sanmuw** [jd-opensource/micro-app#342] [General]
每次跳转子应用 都会 
Disconnected from the devServer, trying to reconnect...

**Comment by sanmuw** [comment]
<img width="1913" alt="image" src="https://user-images.githubusercontent.com/22683369/160754938-7b4ea84e-b172-436f-9c19-935a46dddb6f.png">
子应用：umi一套
基座：react脚手架
micro-app应用版本：0.8.5
代码截图：
基座
package。json
<img width="487" alt="image" src="https://user-images.githubusercontent.com/22683369/160755217-aac96a4e-943c-456d-a8de-77c5fe8e0c4a.png">

入口文件：
<img width="603" alt="image" src="https://user-images.githubusercontent.com/22683369/160755303-d598aa19-4b81-475f-a42f-806236ba1f65.png">

路由：
<img width="381" alt="image" src="https://user-images.githubusercontent.com/22683369/160755359-3b857fbe-11cd-480f-b78f-4d60a9d503fc.png">

组件：
<img width="469" alt="image" src="https://user-images.githubusercontent.com/22683369/160755413-74d365b1-3ad9-483b-b08d-b62a1a6757ff.png">

config-overrides。js
<img width="524" alt="image" src="https://user-images.githubusercontent.com/22683369/160755541-32cba006-f79b-46c2-acc5-fa3261627707.png">

子应用：
配置文件
<img width="446" alt="image" src="https://user-images.githubusercontent.com/22683369/160755708-fa6de0e0-df78-4b69-aacb-3914218b33a8.png">


问题：这类问题如何处理？有代码参考吗？以及原理讲解吗？

**Comment by sanmuw** [comment]
@bailicangdu 能帮忙看下吗

**Comment by bailicangdu** [comment]
这个问题是umi的websocket加载地址兜底到基座上，导致失败，下一个版本会解决

**Reply by sanmuw** [reply]
谢谢哈 提供了思考方向

**Reply by jwqStoneatom** [reply]
> 这个问题是umi的websocket加载地址兜底到基座上，导致失败，下一个版本会解决

下一个版本什么时候发布?

**Comment by jwqStoneatom** [comment]
同学, 你这个问题解决了吗, 若解决, 方便po一下解决方案吗

**Reply by sanmuw** [reply]
目前没有从源码的角度去解决 ，我采用的是 用umi兜底umi 的方式 去解决的

**Reply by curr111** [reply]
你好，请问你怎么实现umi兜底umi的，我也遇到类似问题了
